apps/services/temporary-voter-registry-api/migrations/20210512143646-voter-registry.js

```diff
@@ -0,0 +1,51 @@
+'use strict'
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    const transaction = await queryInterface.sequelize.transaction()
+
+    // try is here to make sure this safely fails if index creation fails
+    try {
+      await queryInterface.createTable('voter_registry', {
+        id: {
+          type: Sequelize.INTEGER,
+          autoIncrement: true,
+          primaryKey: true,
+        },
+        national_id: {
+          type: Sequelize.STRING,
+          allowNull: false,
+        },
+        region_number: {
```

Comment:
hvað er region number og name?

Reply:
Region number is a numeric representation of region name assigned by the national registry.
They represent what voting region (kjördæmi) the requested user belongs to